valence_protocol/src/translation_key.rs

```diff
@@ -0,0 +1,3 @@
+include!(concat!(env!("OUT_DIR"), "/translation_key_consts.rs"));
```

Comment:
nit: I would prefer to have these outputs in a single file because we have IDEs that can fold code.